### ISSUE TITLE
ModuleNotFoundError: No module named 'celery'

### ISSUE THREAD

**Issue by heyscam** [ui/django-post_office#304]
Just upgraded a project from 3.3.1 to 3.4.0. When running `process_tasks` I get the error:

```
  File "XXX/.venv/lib/python3.7/site-packages/post_office/tasks.py", line 6, in <module>
    from celery import shared_task
ModuleNotFoundError: No module named 'celery'
```

I could add celery to my project's dependencies, but I'm not actually using it (I'm sending the emails with a different async queue package). Is it a hard dependency now? Post office is awesome BTW, thanks!

**Comment by jrief** [comment]
`celery` is an optional dependency.

The file `post_office/tasks.py` shouldn't be imported by any other module, except `post_office/apps.py`, which however catches import errors.

What is  `process_tasks`, how do you run that?

**Comment by heyscam** [comment]
Ah sorry I forgot for a moment that `process_tasks` is part of the `django-background-tasks` package, not Post Office. I mentioned it there like I expected you to know what it was - apologies :)

I'm not importing `post_office/tasks.py` in my project. I think what's happening here is that `django-background-tasks` has an auto-discovery mechanism that registers anything contained in a `tasks.py` file. Looks like your `tasks.py` (I guess to do with celery) was added in the last release, hence the breakage when I updated. Obviously that's a conflict, but I would imagine the proper solution would be to change how the discovery mechanism in `django-background-tasks` works. I'll raise an issue over there and see if there's any way to customise it. Thanks.
